### PR TITLE
Update to latest theia image including new remote plugins

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Enhance plugin system with delegating of messages to an external endpoint

### What issues does this PR fix or reference?
N/A

remote-plugins
#### Docs PR

Change-Id: I79f00394d5d45c5e0158f00613ba0c4da64a5530
Signed-off-by: Florent BENOIT <fbenoit@redhat.com>

### CODE REVIEW

**Comment by benoitf** [comment]
ci-build

**Comment by benoitf** [comment]
ci-build

**Comment by benoitf** [comment]
ci-build

File: dockerfiles/theia/e2e/cypress/integration/theia/typescript.spec.ts [hunk 1]
```diff
@@ -86,9 +86,8 @@ context('TypeScript', () => {
             })
 
         }).then(() => {
-            // now, search the word and check there is an error
-            cy.get('.lines-content.monaco-editor-background').contains('invalid').should('have.class', 'mtk15');
-
+            // now, search if there is an invalid syntax
```
Comment (benoitf) [inline dockerfiles/theia/e2e/cypress/integration/theia/typescript.spec.ts]: with new theia/monaco is using overlays to define syntax highlight reports

File: dockerfiles/theia/Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@
 FROM centos
 
 ARG GITHUB_TOKEN
-ARG THEIA_VERSION=0.3.14
+ARG THEIA_VERSION=plugin-id
```
Comment (riuvshin) [inline dockerfiles/theia/Dockerfile:17]: what is `plugin-id` ?

Comment (benoitf) [reply dockerfiles/theia/Dockerfile:17]: @riuvshin the name of the branch (like 0.3.14)

**Review by riuvshin — APPROVED** [review]

**Pull request merged**